packages/react/src/component-tests/IcToast/IcToastTestData.tsx

```diff
@@ -187,7 +187,7 @@ export const MultilineMessageToast = () => {
         <IcToast
           heading="Your coffee is ready"
           ref={toastEl}
-          message="Please dismiss and come to collect your delicious coffee from the barista right away before it gets cold. Dont delay! We wouldn't want you to lose out on your fantastic coffee now would we?"
+          message="Please dismiss and come to collect your delicious coffee from the barista right away before it gets cold. Don't delay! We wouldn't want you to lose out on your fantastic coffee now would we?"
```

Comment:
Currently, the Cypress test suite is failing due to this change. The [IcToast multi-line visual regression image](https://github.com/evenstensberg/ic-ui-kit/blob/feat/cspell-linting/packages/react/cypress-image-diff-screenshots/baseline/IcToast.cy.tsx-multiline.png) needs to be regenerated to include the correct spelling. 

Please delete the image and then run `npm run test-cypress:ci` in the root of the project to regenerate the image.

Please leave a message if you have any issues regenerating the image

Reply:
I’m very limited to commiting anything to the cypress files, because all the test cases visually fail locally. If I can’t fix it by Monday, you could just push to this branch

Reply:
I've spent sometime to research why the baseline images aren't passing. The reason is that each machine creates its own version of the baseline, and will always differ unless the baseline is generated from the exact same setup each time. 

For this, it means that the CI needs to update the baseline images, and it will fail locally despite how I configure cypress. Some people advice to create a common docker container that generates and tests the baseline images against changed ones, so that would also be my advice to you.

Going to try making the CI on my fork to generate new baseline images, and will let you know if I'm stuck.

Reply:
Ah ok. Don't worry about regenerating the image in this case, please delete the [IcToast multi-line visual regression image](https://github.com/evenstensberg/ic-ui-kit/blob/feat/cspell-linting/packages/react/cypress-image-diff-screenshots/baseline/IcToast.cy.tsx-multiline.png) and we'll fix it on our end

Reply:
Done! :)

Reply:
Please merge the new React commit into the existing one

Reply:
@GCHQ-Developer-112 have a look

Reply:
PTAL @GCHQ-Developer-112

Reply:
@GCHQ-Developer-112 PTAL